04-onion-routing.md

```diff
@@ -290,13 +295,11 @@ The reader:
   - If `encrypted_recipient_data` is present:
     - If `blinding_point` is set in the incoming `update_add_htlc`:
       - MUST return an error if `current_blinding_point` is present.
-      - MUST use that `blinding_point` as the blinding point for decryption.
```

Comment:
Why did you remove this? `blinding_point` needs to be used as $E_i$ for the Route Blinding Requirements section.

Reply:
Yes, but it's not used as a blinding point.  I changed this and the next one:

```
    - If `blinding_point` is set in the incoming `update_add_htlc`:
      - MUST return an error if `current_blinding_point` is present.
	  - MUST use `blinding_point` as $`E_i`$
    - Otherwise:
      - MUST return an error if `current_blinding_point` is not present.
      - MUST use `current_blinding_point` as $`E_i`$
      - SHOULD add a random delay before returning errors.
    - MUST return an error if `encrypted_recipient_data` does not decrypt to a valid `encrypted_data_tlv` as described in [Route Blinding](#route-blinding).
```
?